crates/walrus-service/daemon_openapi.yaml

```diff
@@ -312,6 +321,160 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Status'
+  /v1/quilts:
+    put:
```

Comment:
nit: I think publisher is using PUT where it should use POST. https://datatracker.ietf.org/doc/html/rfc7231#section-4.3.3 PUT is most commonly used when you know the target URI for the resource and you are creating or updating it. In this case, we are not using a URI for the quilt (or blob), we're just posting data to this collection endpoint, and relying on the semantics of the endpoint to do what it will. (ie: after this PUT, we can't run GET on the same uri and get back our resource)

This is definitely a nit, but maybe in a future version of this API we can change these methods to use POST.

Reply:
Let's not worry about it for now. I don't want to hold things up for this nit.